### PR TITLE
Country-specific "Resources" page

### PR DESCRIPTION
**Pull request by claudiovallejo**
**Story card:** [ch2054](https://app.clubhouse.io/simpledotorg/story/2054/country-specific-resources-page)

## Because

1. We should display country-specific tech team office hours info
2. Each country should see the right content (some countries have unique training material)

## This addresses

1. Each country has it's own "Tech team office hours" card with the respective Zoom link
2. Each country has it's own "Simple Training Guide" presentation
3. The IHCI one-pager is only displayed in India
4. All external links are constants in `ExternalLinksHelper`
5. All Hindi content is only display in India

**India office hours, training presentation, and 1-pager sections**

image
image

**Bangladesh office hours, training presentation, and 1-pager sections**

image
image

**Ethiopia office hours, training presentation, and 1-pager sections**

image
image

### CODE REVIEW

File: app/views/resources/index.html.erb [hunk 1]
```diff
@@ -30,29 +38,37 @@
...
-
+                    <% if CountryConfig.current[:name] == "Ethiopia" %>
+                      <div class="resource-link resource-presentation">
+                          <a href="https://docs.google.com/presentation/d/1GM57_sHZ53huaMCi1Vsg5O2aAjtXKWdLGvsvKzMROME/edit" class="resource-url">
+                              Guide for trainers and program managers
+                              <span>Use this presentation to learn about the Simple program and how to use the Android App and Dashboard</span>
+                              <i class="fas fa-chalkboard-teacher"></i>
+                          </a>
+                      </div>
+                    <% else %>
```
Comment (timcheadle) [inline app/views/resources/index.html.erb:49]: I'd move these sections to partials as well

File: app/views/resources/index.html.erb [hunk 2]
```diff
@@ -190,22 +182,13 @@
...
-                            1-pager
-                            <span>Regularly updated document explaining Simple.</span>
-                            <i class="far fa-file-pdf"></i>
-                        </a>
-                    </div>
+                <h2>
+                  1-pagers about Simple
+                </h2>
+                <% if CountryConfig.current[:name] == "India" %>
+                  <%= render "shared/one_pager_ihci" %>
```
Comment (rsanheim) [inline app/views/resources/index.html.erb:189]: maybe call it `one_pager_india` to be consistent?

File: app/views/shared/_one_pager_simple.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="resource-link resource-doc">
+  <a
+    href="https://drive.google.com/open?id=1LzGcSK5nPS0izJT4kV7B3y6syl84ACqo&authuser=dburka%40resolvetosavelives.org&usp=drive_fs"
```
Comment (rsanheim) [inline app/views/shared/_one_pager_simple.html.erb:3]: These static doc references would be worth pulling out to a constant or app helper...maybe something like `OnePageHelper` that has the links for various docs.  Its nice to keep static stuff like this consolidated in one file for when the URLs change in the future.

**Review by rsanheim — APPROVED** [review]
approved - left some optional feedback

**Review by rsanheim — APPROVED** [review]
Looks good!

You can run the following to auto fix some of the lint errors:

```
bundle exec standardrb --fix
```

**Pull request merged**